tests/e2e/e2e_test.go

```diff
@@ -211,3 +212,33 @@ func (s *IntegrationTestSuite) TestStateSync() {
 	err = chain.RemoveNode(stateSynchingNode.Name)
 	s.NoError(err)
 }
+
+func (s *IntegrationTestSuite) TestExpeditedProposals() {
+	if !s.skipUpgrade {
+		s.T().Skip("this can be re-enabled post v12")
```

Comment:
Let's make an issue and link it here with a comment please

Reply:
https://github.com/osmosis-labs/osmosis/issues/2386, included this in the same issue as above, will likely take this post v12 since I have context